resources/hg-ReferralComposition.xml

```diff
@@ -1,556 +1,556 @@
-      <alias value="Context" />
-      <mapping>
-        <identity value="ART-DECOR-hg-Kern" />
-        <map value="hg-dataelement-1710" />
-        <comment value="Context" />
-      </mapping>
-    </element>
-    <element id="Composition.section:messageReason.title">
-      <path value="Composition.section.title" />
-      <example>
-        <label value="Example for referral" />
-        <valueString value="Reden bericht" />
-      </example>
-    </element>
-    <element id="Composition.section:messageReason.code">
-      <path value="Composition.section.code" />
-      <min value="1" />
-      <patternCodeableConcept>
-        <coding>
-          <system value="http://snomed.info/sct" />
-          <code value="440378000" />
-        </coding>
-      </patternCodeableConcept>
-    </element>
-    <element id="Composition.section:messageReason.entry">
-      <path value="Composition.section.entry" />
-      <slicing>
-        <discriminator>
-          <type value="value" />
-          <path value="$this" />
-        </discriminator>
-        <rules value="open" />
-      </slicing>
-    </element>
-    <element id="Composition.section:messageReason.entry:problem">
-      <path value="Composition.section.entry" />
-      <sliceName value="problem" />
-      <short value="Problem" />
-      <definition value="Een probleem beschrijft een toestand met betrekking tot de gezondheid en/of het welzijn van een individu. Deze toestand kan zijn benoemd door de patiënt zelf (een klacht), of door zijn of haar zorgverlener (onder andere een diagnose)." />
-      <alias value="Probleem" />
-      <max value="1" />
-      <type>
-        <code value="Reference" />
-        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem" />
-      </type>
-      <mapping>
-        <identity value="ART-DECOR-hg-Kern" />
-        <map value="hg-dataelement-1886" />
-        <comment value="Probleem" />
-      </mapping>
-    </element>
+        <element id="Composition.section:messageReason">
+            <path value="Composition.section" />
+            <sliceName value="messageReason" />
+            <label value="RedenBericht" />
+            <short value="MessageReason" />
+            <definition value="Geeft de reden van de verwijzing of de update van een huisarts naar een paramedicus.&#xD;&#xA;Hierbij is de beschrijving als vrije tekst op aangeven van het NHG verplicht.&#xD;&#xA;Daarnaast kan er ook een ICPC-code van de episode worden meegestuurd, al dan niet aangevuld met meer details over de vastlegging van de ICPC." />
```

Comment:
You might consider making explicit in which way the ICPC code (and additional details) are to be exchanged, by adding a comment on the problem `entry` (or do we think that is too obvious?).

Reply:
And if we conclude that ICPC is indeed mandatory in case a Problem is used to convey the reason of the message (see discussion in ELZT-61), should we also add that on the aforementioned `entry`?

Reply:
@LuudSlagter agreed, are you willing to do a proposal?